build-system/test-configs/config.js

```diff
@@ -108,8 +108,8 @@ const devDashboardTestPaths = ['build-system/server/app-index/test/**/*.js'];
 const jisonPath = 'extensions/**/*.jison';
 
 const lintGlobs = [
-  '**/*.js',
-  '**/*.ts',
+  '**/*.js{,x}',
+  '**/*.ts{,x}',
```

Comment:
nit: I've been bitten by doing this before and accidentally dropping the comma at some point. Sometimes I like this pattern, but here I'd maybe be more explicit
```suggestion
  '**/*.{js,jsx}',
  '**/*.{ts,tsx}',
```

Alternately, if we like being concise, there's always `'**/*.{j,t}s{,x}'` 🙃

Reply:
this is a rare case where I hate being concise